components/button/button.css

```diff
@@ -2,10 +2,310 @@
 
 .btn-title {
   font-size: 3rem;
-  padding: 1rem;
+  /* padding: 1rem; */
 }
 
 .btn-content {
   font-size: 2rem;
+  padding: 5px;
+}
+
+.btn {
   padding: 1rem;
+  cursor: pointer;
+  gap: 1rem;
+  max-width: fit-content;
+}
+
+.btn-primary-container {
+  margin: 2rem;
+  width: 70%;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  position: relative;
+  box-sizing: border-box;
+  border: 2px solid var(--black);
+}
+
+.btn-primary {
+  margin: 2rem;
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  position: relative;
+  font-weight: 500;
+  font-size: 2rem;
+  background-color: var(--primary-color);
+  border: 2px;
+  border-radius: 4px;
+  text-transform: uppercase;
+  transition: background-color 0.2s ease;
+}
+
+.btn-primary:hover {
+  background-color: var(--secondary-color);
+}
+
+.btn-primary-outline {
+  display: inline-flex;
+  justify-content: center;
+  position: relative;
+  box-sizing: border-box;
+  border: 0px;
+  margin: 0px;
+  cursor: pointer;
+  text-decoration: none;
+  background-color: transparent;
+  font-size: 1.5rem;
+  text-transform: uppercase;
+  border: 1px solid black;
+  transition: background-color 0.25s ease-in;
+}
+
+.btn-primary-text {
+  display: inline-flex;
+  justify-content: center;
+  position: relative;
+  box-sizing: border-box;
+  border: 0px;
+  margin: 0px;
+  cursor: pointer;
+  text-decoration: none;
+  background-color: transparent;
+  font-size: 1.5rem;
+  text-transform: uppercase;
+  transition: background-color 0.25s ease-in;
+  color: var(--primary-color);
+}
+
+.btn-link {
+  display: inline-flex;
+  justify-content: center;
+  position: relative;
+  box-sizing: border-box;
+  border: 0px;
+  margin: 0px;
+  cursor: pointer;
+  text-decoration: none;
+  background-color: transparent;
+  font-size: 1.5rem;
+  text-transform: uppercase;
+  transition: background-color 0.25s ease-in;
+  border-radius: 4px;
+  color: var(--primary-color);
+}
+
+.btn-link:hover {
+  color: var(--black);
+  background-color: var(--primary-color);
+}
+
+.btn-secondary-container {
+  margin: 2rem;
+  width: 70%;
+  display: flex;
+  justify-content: space-around;
+  align-items: center;
+  position: relative;
+  box-sizing: border-box;
+  border: 2px solid var(--black);
+  background-color: var(--primary-color);
+}
+
+.btn-secondary {
+  margin: 2rem;
```

Comment:
repetative code this could be mentioned in a class and used for all buttons

Reply:
fixed